Source/WebCore/bindings/js/SerializedScriptValue.cpp

```diff
@@ -691,7 +691,7 @@ static unsigned countUsages(CryptoKeyUsageBitmap usages)
     return count;
 }
 
-enum class CryptoKeyOKPOpNameTag {
+enum class CryptoKeyOKPOpNameTag : bool {
```

Comment:
uint8_t as this will most likely need more than 2 values in future. `x448` and `ed448`

Reply:
Ditto; we could extend it as needed later.